src/components/create-user.html

```diff
@@ -1,95 +1,136 @@
+        width: 20%;
+        margin: auto;
+        margin-top: 25px;
+        padding-bottom: 25px;
+      }
 
-            button:hover {
-                background-color: rgb(196, 226, 245);
-            }
-        </style>
+      .button {
+        background-color: #87CEFA;
+        border: none;
+        color: white;
+        padding: 5px 12px;
+        text-align: center;
+        text-decoration: none;
+        display: block;
+        margin: auto;
+        font-size: 16px;
+        margin-top: 25px;
+        border-radius: 8px;
+      }
+
+      button:hover {
+        background-color: rgb(196, 226, 245);
+      }
 
+      .container {
+        padding: 10px 16px;
+        width: 80%;
+      }
+
+      .container::after,
+      .row::after {
+        content: "";
+        clear: both;
+        display: table;
+      }
+
+      .card {
+        box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+        padding-top: initial;
+      }
+    </style>
+
+    <div class="card">
+      <div class="container">
         <h1 class="div-center">Create User</h1>
+        <form>
+          <label for="fname">First Name</label>
+          <input type="text" placeholder="Your name.." value="{{user.firstName::input}}">
+
+          <label for="lname">Last Name</label>
+          <input type="text" placeholder="Your last name.." value="{{user.lastName::input}}">
+
+          <label for="phone">Phone</label>
+          <input type="text" placeholder="Your phone number.." value="{{user.phone::input}}">
+
+          <label for="email">Email</label>
+          <input type="text" placeholder="Your email.." value="{{user.email::input}}">
+        </form>
         <div>
-            <div class="container">
-                <div class="form-border">
-                    <form>
-                        <label for="fname">First Name</label>
-                        <input type="text" id="fname" name="firstname" placeholder="Your name..">
-
-                        <label for="lname">Last Name</label>
-                        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
-
-                        <label for="phone">Phone</label>
-                        <input type="text" id="phone" name="phone" placeholder="Your phone number..">
-
-                        <label for="email">Email</label>
-                        <input type="text" id="email" name="email" placeholder="Your email..">
-                        <div>
-                            <button class="button">Create</button>
-                        </div>
-                    </form>
-                </div>
-            </div>
+          <button class="button" on-click="createUser">Create User</button>
         </div>
-    </template>
+      </div>
+    </div>
+  </template>
 
-    <script type="module">
+  <script type="module">
+    import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { LocalStorage } from '../api/localStorage.js';
```

Comment:
this should be passed from `intern-app`, or whatever your most parent component is. There really shouldn't be a need to import  your LocalStorage api anywhere but `intern-app`